presto-main/src/main/java/io/prestosql/testing/DateTimeTestingUtils.java

```diff
@@ -43,6 +45,13 @@ public static SqlTimestamp sqlTimestampOf(
         return sqlTimestampOf(precision, LocalDateTime.of(year, monthOfYear, dayOfMonth, hourOfDay, minuteOfHour, secondOfMinute, millisToNanos(millisOfSecond)));
     }
 
+    public static SqlTimestamp sqlTimestampOf(int precision, long epochSeconds, int epochNanos)
```

Comment:
I do not see a benefit of adding this method vs using `SqlTimestamp.newInstance(precision, epochMicros, picosOfMicro)` directly in test